ext/snmp/tests/gh16959.phpt

```diff
@@ -25,6 +25,11 @@ try {
 
 try {
 	snmp2_set($hostname, $communityWrite, $bad_object_ids, array(new stdClass()), array(null));
+} catch (Throwable $e) {
+	echo $e->getMessage() . PHP_EOL;
+}
+try {
+	snmp2_set($hostname, $communityWrite, $bad_object_ids, array("toolongtype"), array(null));
```

Comment:
Please also add a test with a non-interned string.
A trick to make one is using `str_repeat("my string", random_int(1, 1))`.